config/brokerSets.json

```diff
@@ -0,0 +1,9 @@
+{
+  "brokerSets":[
```

Comment:
We should probably add this file name as the default value of broker.set.config.file in cruisecontrol.properties. Otherwise the build will always fail without this value provided:

   ```
 org.apache.kafka.common.config.ConfigException: Configuration broker.set.config.file must be provided.
        at com.linkedin.kafka.cruisecontrol.KafkaCruiseControlUtils.getRequiredConfig(KafkaCruiseControlUtils.java:165)
        at com.linkedin.kafka.cruisecontrol.config.BrokerSetFileResolver.configure(BrokerSetFileResolver.java:52)
        at com.linkedin.kafka.cruisecontrol.config.KafkaCruiseControlConfig.getConfiguredInstance(KafkaCruiseControlConfig.java:81)
        at com.linkedin.kafka.cruisecontrol.analyzer.BalancingConstraint.<init>(BalancingConstraint.java:85)

```

For users who don't need this feature, it seems pretty confusing.

Reply:
Same as default.broker.set.assignment.policy.class. There should be a default value

Reply:
Fixed